--- target/linux/sunxi/image/Makefile.original	2021-02-27 06:19:22.815379900 +0300
+++ target/linux/sunxi/image/Makefile	2021-02-27 06:20:00.557396600 +0300
@@ -17,6 +17,10 @@
 	rm -f $@.boot
 	mkfs.fat $@.boot -C $(FAT32_BLOCKS)
 
+	TMP_HOSTNAME_PART=$( shell eval 'tr -dc A-Z </dev/urandom | head -c6' )
+	echo "kernel.hostname=OPIZVH" | mcopy -i $@.boot - ::hostname.txt
+	echo "ServerName=OPIZVH" | mcopy -i $@.boot - ::vhusbd.ini
+	undefine TMP_HOSTNAME_PART
 	mcopy -i $@.boot $(STAGING_DIR_IMAGE)/$(DEVICE_NAME)-boot.scr ::boot.scr
 	mcopy -i $@.boot $(DTS_DIR)/$(SUNXI_DTS).dtb ::dtb
 	mcopy -i $@.boot $(IMAGE_KERNEL) ::uImage
